### PR TITLE
fix: make coordinator exit gracefully

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/workflow/coordinator/main.go [hunk 1]
```diff
@@ -23,6 +30,7 @@ func main() {
 	var err error
 	var message string
 	defer func() {
+		time.Sleep(ExitDelayTime)
```
Comment (cd1989) [inline cmd/workflow/coordinator/main.go]: Please describe why need this delay in comment.

File: cmd/workflow/coordinator/main.go [hunk 2]
```diff
@@ -12,6 +13,12 @@ import (
 	"github.com/caicloud/cyclone/pkg/workflow/coordinator"
 )
 
+const (
+	//  ExitDelayTime is the delay time of coordinator exit, coordinator need the time to do
+	// something, like collecting logs, to make exit gracefully
+	ExitDelayTime = 3 * time.Second
```
Comment (supereagle) [inline cmd/workflow/coordinator/main.go]: Change to `exitDelayTime` as there is no need to export it.

**Pull request merged**